www/docs/customization/snapcraft.md

```diff
@@ -60,12 +60,21 @@ snapcrafts:
     # store.
     description: This is the best drum roll application out there. Install it and awe!
 
+    # Channels in store where snap will be pushed.
+    # Default depends on grade.
```

Comment:
probably worth saying here which channels are default for which grades?

Reply:
Grade say something about it, but we may be more explicit indeed :)